packages/govuk-frontend/tasks/scripts.mjs

```diff
@@ -48,5 +48,8 @@ export const compile = (options) => gulp.series(
       srcPath: join(options.srcPath, 'govuk-prototype-kit'),
       destPath: resolve(options.destPath, '../') // Top level (not dist) for compatibility
     })
-  )
+  ),
+
+  // Compile GOV.UK Frontend build stats
+  npm.script('build:stats', [], options)
```

Comment:
Why does govuk-frontend need to run the stats itself? I was picturing the stats as completely independent and us coordinating its run in a similar  way we coordinate the run of the review app.

Reply:
It's what keeps the review app stats up-to-date as you make changes

Otherwise they're only generated at the start of the session

It was an easy fix versus moving to `npx rollup --watch`

Reply:
Oh, I see. I thought we discussed that `watch` wasn't in scope, but that looks like a very cheap option to achieve it 🙌🏻

Reply:
Yeah exactly, hope that explains it—bonus feature!